test/e2e/config/azure-dev.yaml

```diff
@@ -97,8 +99,9 @@ variables:
   EXP_MACHINE_POOL: "true"
   EXP_CLUSTER_RESOURCE_SET: "true"
   CONFORMANCE_CI_ARTIFACTS_KUBERNETES_VERSION: "v1.19.7"
-  CONFORMANCE_WORKER_MACHINE_COUNT: "5"
+  CONFORMANCE_WORKER_MACHINE_COUNT: "2"
   CONFORMANCE_CONTROL_PLANE_MACHINE_COUNT: "1"
```

Comment:
We should test with 3 control plane nodes unless the additional "transition from 1 to 2 nodes to 3 nodes" foo is not really relevant to testing any possible set of upstream k/k commits.

cc @CecileRobertMichon @devigned

Reply:
Looks like it's addressed by https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1329